core/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -126,7 +128,7 @@ public static ThreadPoolType fromType(String type) {
         map.put(Names.GET, ThreadPoolType.FIXED);
         map.put(Names.INDEX, ThreadPoolType.FIXED);
         map.put(Names.BULK, ThreadPoolType.FIXED);
-        map.put(Names.SEARCH, ThreadPoolType.FIXED);
+        map.put(Names.SEARCH, ThreadPoolType.FIXED_AUTO_QUEUE_SIZE);
```

Comment:
so who we use the new `FIXED_AUTO_QUEUE_SIZE` but if somebody had some custom settings on their search threadpool will they still work? I mean are the settings for `FIXED` a true subset of the one in `FIXED_AUTO_QUEUE_SIZE`?